app/src/main/java/edu/ucsc/cmps115_spring2017/face2name/Layer/LayerView.java

```diff
@@ -0,0 +1,62 @@
+package edu.ucsc.cmps115_spring2017.face2name.Layer;
+
+/**
+ * Created by Chris Myau on 4/25/2017.
+ */
+
+import android.content.Context;
+
+import android.util.AttributeSet;
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.graphics.Rect;
+import android.view.View;
+import 	android.view.MotionEvent;
+
+
+public final class LayerView extends View
+{
+    private Paint paint;
+    int color;
```

Comment:
Use the `mMemberName` style of naming. That will make it consistent with the rest of the codebase.